sourceaddrs/source_registry_final.go

```diff
@@ -0,0 +1,64 @@
+package sourceaddrs
+
+import (
+	"github.com/apparentlymart/go-versions/versions"
```

Comment:
Post-hoc comment, but this is the first time I am seeing this - why not hashicorp/go-version? We use that regularly throughout the rest of TFC's codebases.

Reply:
We use this library in Terraform because `go-version` tends to generate useless error messages and historically it didn't handle prereleases correctly, although I think that latter point may at least have been fixed in the meantime. But consistency with what Terraform itself does is the main thing here, so better to use the library it uses.